docs/helm/README.md

```diff
@@ -1,3 +1,21 @@
-# Using Helm Charts
+# Using Helm charts
```

Comment:
Didn't we choose to use "Title Case" for headers like this?

Reply:
Okay...I remember seeing the opposite PR, but never check was it merged -> will change. Thanks for pointing that out.

Reply:
Done